client/components/PhotoGrid.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function PhotoGrid({ photos }) {
+  // Load up to 5 photos for photo cover
+  const maxPhotos = photos.length < 5 ? photos.length : 5;
+  const coverPhotos = [
+    <div className="cover-item cover-item-main" key="0">
```

Comment:
Do you have anything that will set these CSS names to something local? Styled Components or CSS Modules?

We need to be careful to not cross-contaminate our CSS namespaces.

Reply:
I have a ticket open to investigate Style Components. Thanks